### ISSUE TITLE
Error with arbitrary commands when run-count is greater than 1

### ISSUE THREAD

**Issue by BonjourGit** [RedisLabs/memtier_benchmark#104]
When arbitrary command (simple set command) is used and run-count is set to be greater than 1, only the first run succeeds and subsequent runs (> 1) were found to be failing.
Redis server was replying with "-ERR unknown command" response.
Examining the TCP dump of the redis commands sent by memtier_benchmark suggests that the RESP was malformatted for runs > 1.

**Test Setup**
memtier_benchmark: master branch @f55b4b6
Redis: 5.0.7, standalone
Runs on the same host

**To reproduce the issue**
```
memtier_benchmark -s localhost -p 6379 --protocol=redis --clients=1 --run-count=3 --command="set __key__ __data__"
```
**Log**
```
[RUN #1] Preparing benchmark client...
[RUN #1] Launching threads now...
[RUN #1 74%,   1 secs]  4 threads:       29582 ops,   29573 (avg:   29573) ops/sec, 2.22MB/sec (avg: 2.22MB/sec),  0.13 (avg:  0.13) msec latency^M[RUN #1 100%,   1 secs]  0 threads:       40000 ops,   29573 (avg:   29831) ops/sec, 2.22MB/sec (avg: 2.24MB/sec),  0.13 (avg:  0.13) msec latency^M

[RUN #2] Preparing benchmark client...
[RUN #2] Launching threads now...
server 127.0.0.1:6379 handle error response: -ERR unknown command `*3  $3  set  `, with args beginning with: `memtier-6263819`, `xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`,
server 127.0.0.1:6379 handle error response: -ERR unknown command `*3  $3  set  `, with args beginning with: `memtier-6263819`, `xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`,
server 127.0.0.1:6379 handle error response: -ERR unknown command `*3  $3  set  `, with args beginning with: `memtier-6263819`, `xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`,
... Repeated ...
[RUN #2 100%,   1 secs]  0 threads:       40000 ops,   24478 (avg:   23277) ops/sec, 4.80MB/sec (avg: 4.57MB/sec),  0.14 (avg:  0.15) msec latency^M

[RUN #3] Preparing benchmark client...
[RUN #3] Launching threads now...
server 127.0.0.1:6379 handle error response: -ERR unknown command `*3  $13  *3  $3  set    `, with args beginning with: `memtier-6263819`, `xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`,
server 127.0.0.1:6379 handle error response: -ERR unknown command `*3  $13  *3  $3  set    `, with args beginning with: `memtier-6263819`, `xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`,
server 127.0.0.1:6379 handle error response: -ERR unknown command `*3  $13  *3  $3  set    `, with args beginning with: `memtier-6263819`, `xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`,
... Repeated ...
[RUN #3 100%,   1 secs]  0 threads:       40000 ops,   24587 (avg:   24110) ops/sec, 5.34MB/sec (avg: 5.24MB/sec),  0.14 (avg:  0.14) msec latency^M

```
**RESP captured for run>1**

=== Request ===

![image](https://user-images.githubusercontent.com/34569855/70584487-516c9300-1bfc-11ea-8f33-fdfbc50725ad.png)

=== Response ===

![image](https://user-images.githubusercontent.com/34569855/70584536-74974280-1bfc-11ea-940d-9aae271555f9.png)

**Comment by yaacovhazan-Redislabs** [comment]
Thank you @BonjourGit for raising the issue... I create a PR to fix it - fix issue #104 arbitrary command with run count

**Comment by BonjourGit** [comment]
@yaacovhazan-Redislabs 
Thank you for the fix. It is working!
